Update your ALSOs

ooffordable opened the pull request:
by oooffordable

ooffordable committed:
Update README.md

jaywonchung commented on README.md:
```diff
@@ -1,3 +1,8 @@
+An undergraduate interested in **software systems and deep learning**. Also a fingerstyle guitar player. 
+
+
+Also a main TA of [OSSPR2021](https://github.com/swsnu/osspr2021).
```
a -> the please

jaywonchung commented on README.md:
```diff
@@ -1,3 +1,8 @@
+
+
+Also a main TA of [OSSPR2021](https://github.com/swsnu/osspr2021).
+Also a member of the blood-forged ally of [CRANE](https://github.com/snuspl/crane)
```
Missing period please

jaywonchung commented on README.md:
```diff
@@ -1,3 +1,8 @@
+
+Also a main TA of [OSSPR2021](https://github.com/swsnu/osspr2021).
+Also a member of the blood-forged ally of [CRANE](https://github.com/snuspl/crane)
+Also a nvim spreader in SPLab.
```
```markdown
Also a spreader of the Neovim plague in [SPLab](https://spl.snu.ac.kr).
```
please

jaywonchung reviewed (changes requested):
Thank you for your contribution. I left some change requests. Take a look!

ooffordable committed:
Update README.md

ooffordable commented:
> Thank you for your contribution. I left some change requests. Take a look!

Thanks for your timely review. I addressed all of your comments.
Would you please merge my modification to your master branch?

jaywonchung reviewed (approved):
LGTM! I believe that these ALSO statements and you being the contributor summarize my time at SPL very well. I sincerely appreciate your contribution.
